README.md

```diff
@@ -191,13 +107,31 @@ See test README for test file structure.
   Run methods being tested through system tests only because they depend on, and affect the overall system.
   Constructors cannot be tested without getters which most classes don't implement.
 
+### Iteration 2
+- The scheduler algorithm prioritizes the prevention of starvation over achieving
+the highest throughput possible.
+
+  - Throughout: A measure of efficiency. The rate at which the elevator control system can process
+passenger destination requests within a specified amount of time. 
+
+  - Starvation: A passenger can experience starvation if they are left waiting for an elevator
+for an extended period of time. This would happen if they are on a floor that is rarely used and far away.
+A system that favours throughput may only focus on servicing passengers that are close together to increase 
+the amount of requests that can be serviced. 
+
+- Assumption 1: If the design can lead to starvation, it is an invalid design. 
```

Comment:
I think the rest of the section is good enough, this is duplication. I do like this line, it can be added to the top (line 111): If the design can lead to starvation, it is an invalid design.